susemanager-utils/testing/docker/scripts/schema_migration_test_pgsql-31to41.sh

```diff
@@ -60,17 +60,17 @@ fi
 
 # guessing the link to next major version
 # THE NEXT BLOCK CAN BE COMMENTED OUT WHEN WE HAVE THE LINK PACKAGED
-for v in `seq 30 -1 1`; do
-    minusone=$(($v-1))
-    if [ -d /etc/sysconfig/rhn/schema-upgrade/susemanager-schema-4.0.$minusone-to-susemanager-schema-4.0.$v ]; then
-        if [ ! -d /etc/sysconfig/rhn/schema-upgrade/susemanager-schema-4.0.$v-to-susemanager-schema-4.1.0 ]; then
-            mkdir /etc/sysconfig/rhn/schema-upgrade/susemanager-schema-4.0.$v-to-susemanager-schema-4.1.0
-            # set hard this destination
-            #export SUMA_TEST_SCHEMA_VERSION="4.0.1"
-        fi
-        break
-    fi
-done
+#for v in `seq 30 -1 1`; do
```

Comment:
I am not sure I remember what "# THE NEXT BLOCK CAN BE COMMENTED OUT WHEN WE HAVE THE LINK PACKAGED" means... the link between major versions?

If so, we now already have that, so maybe the whole section can go away?

Reply:
If we don't need this code, I would vote on dropping it instead of commenting it, unless you have a good reason to leave it comment it.

Reply:
removed